daphne_server/docker/example-service.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM rust:1.75-bookworm AS builder
+
+RUN apt update && \
+    apt install -y \
+        capnproto
+
+WORKDIR /dap
+
+COPY Cargo.toml Cargo.lock .
+COPY daphne daphne
+COPY daphne_service_utils daphne_service_utils
+COPY daphne_server daphne_server
+RUN cargo new --lib daphne_worker
+RUN cargo new --lib daphne_worker_test
```

Comment:
As I understand it, creating this empty lib is necessary because the workspace defines these crates. This is a little awkwward -- it would be nice if there was a cleaner way to do this.

Reply:
we either sed script away the specification (tried it, went over badly) or we copy each one over just for it to be unused or we do this, either way, the number of lines in the dockerfile is the same and this way we can clearly see which ones matter to the build and which ones don't